.github/workflows/triage-priority-bugs.yml

```diff
@@ -0,0 +1,20 @@
+name: Move labelled issues into the Priority bugs column for the Web App Team
+
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  Move_high_priority_issues_to_team_workboard:
+    runs-on: ubuntu-latest
+    if: >
+        contains(github.event.issue.labels.*.name, 'S-Critical') && (contains(github.event.issue.labels.*.name, 'P-High') ||  contains(github.event.issue.labels.*.name, 'P-Medium')) ||
+        contains(github.event.issue.labels.*.name, 'S-Major') && contains(github.event.issue.labels.*.name, 'P-High') ||
+        contains(github.event.issue.labels.*.name, 'A11y') && contains(github.event.issue.labels.*.name, 'P-High')
```

Comment:
Sorry, I am a little confused by what this exactly does - it seems right now that any `P-Medium` gets moved? Could parts of the statement perhaps be wrapped in `()` so that it's a bit clearer what is done first